docs/content/en/docs/getting-started/metrics/_index.md

```diff
@@ -4,3 +4,321 @@ description: Learn how Keptn metrics enhances your deployment
+you can install just the `metrics-operator`
+bu modifying Helm values.
+See
+[Install KLT](../../install/install.md)
+for more information about installing the Lifecycle Toolkit.
+
+To verify that the `metrics-operator` is installed in your cluster,
+run the following command:
+
+```shell
+kubectl get pods -n keptn-lifecycle-toolkit-system
+```
+
+The output shows all components that are running on your system.
+
+## Enable KLT for your cluster
+
+To enable KLT for your cluster, annotate the Kubernetes
+[Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+resource.
+In this example, this is defined in the
+[simplenode-dev-ns.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-ns.yaml)
+file, which looks like this:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: simplenode-dev
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"
+```
+
+You see the annotation line that enables `lifecycle-toolkit`.
+This line tells the webhook to handle the namespace
+
+## Integrate KLT with your cluster
+
+To integrate KLT with your cluster, annotate the Kubernetes
+[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+resource.
+In this example, this is defined in the
+[simplenode-dev-deployment.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-deployment.yaml)
+file, which includes the following lines:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: simplenode
+  namespace: simplenode-dev
+...
+template:
+    metadata:
+      labels:
+        app: simplenode
+        app.kubernetes.io/name: simplenodeservice
+      annotations:
+        # keptn.sh/app: simpleapp
+        keptn.sh/workload: simplenode
+        keptn.sh/version: 1.0.2
+        keptn.sh/pre-deployment-evaluations: evaluate-dependencies
+        keptn.sh/pre-deployment-tasks: notify
+        keptn.sh/post-deployment-evaluations: evaluate-deployment
+        keptn.sh/post-deployment-tasks: notify
+...
+```
+
+For more information about using annotations and labels
+to integrate KLT into your deployment cluster, see
+[Integrate KLT with your applications](../../implementing/integrate.md).
+
+The [Kubernetes metric server](https://github.com/kubernetes-sigs/metrics-server)
+requires that you maintain point-to-point integrations
+from Argo Rollouts, Flux, KEDA, and HPA.
+Each has plugins but it is difficult to maintain them,
+especially if you are using multiple tools
+and multible observability platforms.
+The Keptn Metrics Server unifies and standardizes access to this data.
+
+The steps to implement Keptn metrics are:
+
+## Install and configure KLT
+
+Use the Helm Chart to install the Keptn Metrics Server
+as part of the Lifecycle Toolkit
+or completely stand-alone.
+ See
+[Install KLT using the Helm Chart](../../install/install.md/#use-helm-chart).
+-- End of Probably goes --
```

Comment:
This was already provided few section before

Reply:
Whoops -- that "End of Probably goes -- marker indicates that I meant to delete that.  I'll fix.

Reply:
Fixed